source/use-cases/sensitive-data-encryption.txt

```diff
@@ -75,20 +75,106 @@ velit congue vel.
 D. Perform Encrypted Read/Write Operations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Etiam hendrerit posuere justo, ac hendrerit lorem pellentesque sed.
-Fusce ullamcorper neque eu libero aliquam, sed placerat sapien cursus.
-Vestibulum maximus magna sed velit feugiat tempor.
-
-Ut accumsan pellentesque metus, at porta felis dapibus vulputate. Aenean
-malesuada interdum libero in feugiat. Fusce pharetra massa eget leo
-varius, et gravida massa tempus.
-
-Ut mattis eros elit, sed ultrices purus lacinia sed. Curabitur tincidunt
-tellus eget neque porta dictum ac vel orci. Vestibulum ante ipsum primis
-in faucibus orci luctus et ultrices posuere cubilia Curae.
-
-Praesent feugiat arcu non risus elementum eleifend. Nullam sed iaculis
-erat.
+Read and write operations on encrypted fields use the same methods and have the capabilities of traditional MongoDB read and write operations with the added security of field-level encryption.
+
+MedcoMD engineers continue the development of the Medical Care Management 
+System. In the system, doctors should be able to read and write to all fields, 
+and receptionists should only be allowed to read and write to non-sensitive fields. 
+
+MedcoMD engineers create a function to insert a new patient:
+
+
+.. tabs::
+
+   tabs:
+
+     - id: java-jsonschema-generation
```

Comment:
We will likely address this during the full review, but I think the id needs to be the same as all the other tab so that when we have a second tab (e.g. Node), switching tabs will activate all the Java tabs or all the Node ones. I also think that we're using "tabid" and am not certain whether id provides any functionality in this context. Here is the directive I see in the main doc:
```
:tabid: use-cases-fle-guide-java
```